lib/bemhtml/index.js

```diff
@@ -81,10 +81,7 @@ BEMHTML.prototype.render = function render(context,
 
   var isBEM = bem;
   if (isBEM === undefined) {
-    if (ctx.bem === undefined)
-      isBEM = entity.block || entity.elem;
-    else
-      isBEM = ctx.bem;
+    isBEM = entity.block || entity.elem;
```

Comment:
Because of `bem` in line 82 is result of `bem()` mode execution. And `bem()` returns `this.ctx.bem` by default.